test/e2e/issues.spec.ts

```diff
@@ -469,4 +469,19 @@ describe('Issues', function () {
     const result = engine.parseAndRenderSync('{{ÜLKE}}', { ÜLKE: 'Türkiye' })
     expect(result).toEqual('Türkiye')
   })
+  it('#670 Does not render multiple `else` branches in the same if conditional', () => {
```

Comment:
We can move edge cases into tags/, while preserving the case mentioned in that issue.